science/2021-07-08-science-blog-2/index.md

```diff
@@ -0,0 +1,13 @@
+---
+page-title: "Event-related questioning (1/2) of the Corona-Warn-App"
+page-description: "Event-related questioning (1/2) of the Corona-Warn-App"
```

Comment:
```suggestion
page-title: "Corona-Warn-App: Event-related survey (1/2)"
page-description: "Corona-Warn-App: Event-related survey (1/2)"
```

I would use the word "survey" instead of "questioning".

https://dictionary.cambridge.org/dictionary/english/survey
an examination of opinions, behaviour, etc., made by asking people questions:

"questioning" has a different meaning! 😄 
https://dictionary.cambridge.org/dictionary/english/questioning
the situation in which the police ask someone questions about a crime:

so the sentence means that the police are asking the app about a crime that may have been committed. "of" in that position is also problematic because it is not clear whether the app is being questioned or the user of the app.

That can be solved by changing the order of the words, so that "of" (or some other preposition) isn't necessary.